CONTRIBUTING.md

```diff
@@ -15,7 +15,15 @@ Docker-compose
         DB_NAME=postgres
         PORT=5432
         SSL_MODE=disable
-```      
+``` 
+- Create a `flyway.conf` file in `/flyway/conf`
```

Comment:
these should not be documented. instead, it can be shared by committing that config file into git